### ISSUE TITLE
Document use of "desired" properties together with Eclipse Vorto definitions

### ISSUE THREAD

**Issue by thjaeckle** [eclipse/ditto#981]
The current ["The link to Vorto"](https://www.eclipse.org/ditto/basic-feature.html#the-link-to-eclipse-vorto) documentation of Ditto's "features" do not yet mention the added ["desired" properties](https://www.eclipse.org/ditto/basic-feature.html#feature-desired-properties).

Also think about extracting a new separate doc. page describing "The link to Vorto".

**Comment by w4tsn** [comment]
Maybe this can be generalized a bit since the definition is more or less only the link to the model of a thing and vorto happens to be a first go-to-solution that integrates well with ditto. In theory you could use any other model system if I'm not mistaken. So in general the problem is more about if I want to have a model for my data, how to link it with the thing and what to gain from that, right?

**Comment by thjaeckle** [comment]
Yes, however the ["The link to Vorto"](https://www.eclipse.org/ditto/basic-feature.html#the-link-to-eclipse-vorto) section is about Vorto :D

Maybe we can add some words to the "general" concept of linking to a foreign model description. However, that would be really abstract as we did not ever try out another modeling system than Vorto.
